JavaScript/JavaScriptSDK.Tests/CheckinTests/appInsights.tests.ts

```diff
@@ -655,6 +655,48 @@ class AppInsightsTests extends TestClass {
             }
         });
 
+        this.testCase({
```

Comment:
I'd add another case for null error, and maybe some another exotic case with "", 0, etc instead of error object.